Makefile

```diff
@@ -43,12 +43,18 @@ local:
 check:
 	black --check ./
 	cargo fmt --all -- --check
-	PQ_LIB_DIR=/usr/pgsql-9.6/lib cargo check
+	PQ_LIB_DIR=/usr/pgsql-9.6/lib cargo check --locked --all-targets
```

Comment:
is this to renew `Cargo.lock` file?

Reply:
This checks the code has no errors. The `--locked` flag ensures the Cargo.lock file is up to date. The  `--all-targets` flag ensures the check is run for all targets.

Reply:
Thanks